File: HTML/META-INF/resources/frame/default/SiteMapForm.js
/**
 * メニューフォーム.
 */



/**
 * メニューフォーム.
 */

createSubclass("SiteMapForm", {}, "MenuForm");
/**
 * ページの各エレメントとの対応付け.
 */
SiteMapForm.prototype.attach = function() {
	Form.prototype.attach.call(this);
	this.menu = this.newInstance(this.menu);
	this.menu.init();
	this.menu.attach();
};

File: src/dataforms/app/page/sitemap/SiteMapForm.java
package dataforms.app.page.sitemap;

import dataforms.app.form.MenuForm;
import dataforms.menu.Menu;

/**
 * サイドメニューフォームクラス。
 *
 */
public class SiteMapForm extends MenuForm {
	/**
	 * コンストラクタ。
	 */
	public SiteMapForm() {
		super(null);
	}

	@Override
	protected Menu newMenuComponent() {
		Menu menu = new Menu();
		return menu;
	}
}
